packages/libs/sdk/src/api/controllers/nfts.ts

```diff
@@ -159,7 +159,7 @@ export class NftsController {
   private async getByWalletAddress(
     variables: WalletNFTsByAddressQueryVariablesType & NonQueryInput
   ): Promise<WalletNFTsByAddressFormattedResult> {
-    const { chain, contractAddresses, ...queryVariables } = variables;
+    const { chain, ...queryVariables } = variables;
```

Comment:
This was a small change that I missed from https://github.com/quiknode-labs/qn-oss/commit/42a66271d29eeca6c6ee2fb1c8efc251bc0f2699, I noticed in the intellisense options when testing out the built package